tests/third_party/cupy/manipulation_tests/test_shape.py

```diff
@@ -115,15 +102,15 @@ def test_reshape_zerosize(self, xp):
         return b
 
     @pytest.mark.skip("array.base is not implemented")
-    @testing.for_orders(_supported_orders)
+    @testing.for_orders("CFA")
     @testing.numpy_cupy_array_equal(strides_check=True)
     def test_reshape_zerosize2(self, xp, order):
         a = xp.zeros((2, 0, 3))
         b = a.reshape((5, 0, 4), order=order)
         assert b.base is a
```

Comment:
I think it would be better to unmute the test and to comment `assert b.base is a` line.
The same for `test_reshape_zerosize`